src/pages/public/Companion/components/CompanionLayout.js

```diff
@@ -138,7 +140,7 @@ const CompanionLayout = (params) => {
       landing
     },
     email,
-    setEmail,
+    // setEmail,
```

Comment:
Why are we commenting setEmail?

Reply:
build errors – not in use for this code change